doc/mgmt/SONiC_OC_MGMT_Interface_HLD.md

```diff
@@ -0,0 +1,259 @@
+# Feature Name
+Openconfig support for Management Interface via openconfig-interfaces.yang.
+# High Level Design Document
+#### Rev 0.1
+
+# Table of Contents
+  * [List of Tables](#list-of-tables)
+  * [Revision](#revision)
+  * [Scope](#scope)
+  * [Definition/Abbreviation](#definitionabbreviation)
+
+# List of Tables
+[Table 1: Abbreviations](#table-1-abbreviations)
+
+# Revision
+| Rev |     Date    |       Author       | Change Description                |
+|:---:|:-----------:|:------------------:|-----------------------------------|
+| 0.1 | 09/09/2019  |   Ravi Vasanthm     | Initial version                   |
+
+# About this Manual
+This document provides general information about openconfig support for Management interface handling in SONIC.
+
+# Scope
+This document describes the high level design of openconfig support for Management interface handling feature. Call out any related design that is not covered by this document
+
+# Definition/Abbreviation
+
+### Table 1: Abbreviations
+| **Term**                 | **Meaning**                         |
+|--------------------------|-------------------------------------|
+| MGMT Intf                     | Management Interface                     |
+
+# 1 Feature Overview
+Currently SONIC does not support managing/configuring Management interface parameters via CLI, REST and GNMI. As part of this feature will be providing config set/get and status get support for Management interface via openconfig-interfaces.yang.
```

Comment:
Can just remove the first part in the first sentence, and consolidate this to something like:
"This feature will provide config set/get and status get support for Management interface according to the `openconfig-interfaces.yang` data model via CLI, REST, and gNMI."

Reply:
Also provide a link to the YANG model

Reply:
addressed